gravitee-apim-e2e/ui-test/integration/apim/ui/gateways/ui-gateway-information.spec.ts

```diff
@@ -64,16 +64,16 @@ describe('Get Gateway instance information as admin', () => {
   });
 
   it('should redirect to instance overview showing information of selected gateway', () => {
-    cy.getByDataTestId('instances_instances-box').its(0).trigger('click');
+    cy.getByDataTestId('instances_instances-box').first().find('a').trigger('click');
     cy.url().should('contain', 'environment');
     cy.contains('Information');
     cy.contains('Plugins');
     cy.contains('System properties');
   });
 
   it('should display monitoring information of selected gateway', () => {
-    cy.getByDataTestId('instances_instances-box').its(0).trigger('click');
-    cy.get('[data-testid="management.instances.detail.monitoring"]').trigger('click');
+    cy.getByDataTestId('instances_instances-box').first().find('a').trigger('click');
+    cy.getByDataTestId('management.instances.detail.monitoring').trigger('click');
```

Comment:
It looks like this test is failing after the update 🤔

Reply:
@gaetanmaisse Yes right, one test is failing but I think it was already failing before and I have the impression that it's actually related to a code change in the UI and for this I'd like some help. I will get in contact in Florent in the afternoon to talk about it. If you want and your time allows it, we could also check on that together. Otherwise, as mentioned, I hope to reach Florian later on.